nonspecific_current/neuron/leonhard.cpp

```diff
@@ -190,8 +324,15 @@ namespace neuron {
     }
 
 
-    inline double nrn_current_leonhard(size_t id, leonhard_Instance& inst, leonhard_NodeData& node_data, double v) {
+    inline double nrn_current_leonhard(_nrn_model_sorted_token const& _sorted_token, NrnThread* _nt, Memb_list* _ml_arg, int _type, size_t id, leonhard_Instance& inst, leonhard_NodeData& node_data, double v) {
```

Comment:
Setting up `_lmr` should happen outside the loop. `nrn_current_*` is a function that's only called in the loop body of `nrn_cur_*`.

It might be better to pass it in.